Inc/version.h

```diff
@@ -2,6 +2,6 @@
   update this file for new releases
  */
 #define VERSION_MAJOR 2
-#define VERSION_MINOR 17
+#define VERSION_MINOR 18
```

Comment:
version will only be incremented on release

Reply:
I would rather the release tag in main is bumped after the release. Then anything built on main you know is newer than the last release.

Right now, you build a binary from main, and it has the same 2.17 release name.